{
  "done": [
    "paraview",
    "pyproj",
    "libspatialite",
    "geotiff",
    "gdal",
    "qgis",
    "pygplates",
    "postgis",
    "mapserver"
  ],
  "in-pr": [
    "fimex",
    "rasterio",
    "magics",
    "vtk",
    "magics-metview",
    "magics-metview-batch",
    "r-proj4"
  ],
  "awaiting-pr": [],
  "not-solvable": [
    "r-terra",
    "r-sf",
    "r-rgdal",
    "r-gdalcubes",
    "pynio",
    "ncl"
  ],
  "awaiting-parents": [
    "r-lwgeom",
    "cdo",
    "visan",
    "r-gribr"
  ],
  "bot-error": [],
  "_feedstock_status": {
    "cdo": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "gdal": {
      "num_descendants": 12,
      "immediate_children": [
        "mapserver",
        "ncl",
        "postgis",
        "pygplates",
        "pynio",
        "qgis",
        "r-gdalcubes",
        "r-rgdal",
        "r-sf",
        "r-terra",
        "rasterio"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/gdal-feedstock/pull/744",
      "pr_status": "unknown"
    },
    "geotiff": {
      "num_descendants": 13,
      "immediate_children": [
        "gdal",
        "qgis"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/geotiff-feedstock/pull/54",
      "pr_status": "unknown"
    },
    "libspatialite": {
      "num_descendants": 13,
      "immediate_children": [
        "gdal",
        "qgis"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/libspatialite-feedstock/pull/93",
      "pr_status": "unknown"
    },
    "magics": {
      "num_descendants": 1,
      "immediate_children": [
        "cdo"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/magics-feedstock/pull/121",
      "pr_status": "unstable"
    },
    "ncl": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4780263150\">bot CI job</a>): main: ['linux_64_: Encountered problems while solving:\\n  - package libgdal-3.6.3-h0e7c1b1_3 requires libjpeg-turbo >=2.1.5.1,<3.0a0, but none of the providers can be installed\\n', 'osx_64_: Encountered problems while solving:\\n  - package libgdal-3.6.3-h32fa94e_6 requires libjpeg-turbo >=2.1.5.1,<3.0a0, but none of the providers can be installed\\n', 'osx_arm64_: Encountered problems while solving:\\n  - package libgdal-3.6.3-h0a7289d_7 requires libjpeg-turbo >=2.1.5.1,<3.0a0, but none of the providers can be installed\\n']"
    },
    "paraview": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/paraview-feedstock/pull/181",
      "pr_status": "unknown"
    },
    "postgis": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/postgis-feedstock/pull/93",
      "pr_status": "unknown"
    },
    "pynio": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4780263150\">bot CI job</a>): main: ['linux_64_numpy1.21python3.10.____cpython: Encountered problems while solving:\\n  - package libgdal-3.6.3-h0e7c1b1_3 requires libjpeg-turbo >=2.1.5.1,<3.0a0, but none of the providers can be installed\\n', 'linux_64_numpy1.21python3.8.____cpython: Encountered problems while solving:\\n  - package libgdal-3.6.3-h0e7c1b1_3 requires libjpeg-turbo >=2.1.5.1,<3.0a0, but none of the providers can be installed\\n', 'linux_64_numpy1.21python3.9.____cpython: Encountered problems while solving:\\n  - package libgdal-3.6.3-h0e7c1b1_3 requires libjpeg-turbo >=2.1.5.1,<3.0a0, but none of the providers can be installed\\n', 'linux_64_numpy1.23python3.11.____cpython: Encountered problems while solving:\\n  - package libgdal-3.6.3-h0e7c1b1_3 requires libjpeg-turbo >=2.1.5.1,<3.0a0, but none of the providers can be installed\\n', 'osx_64_numpy1.21python3.10.____cpython: Encountered problems while solving:\\n  - package libgdal-3.6.3-h32fa94e_6 requires libjpeg-turbo >=2.1.5.1,<3.0a0, but none of the providers can be installed\\n', 'osx_64_numpy1.21python3.8.____cpython: Encountered problems while solving:\\n  - package libgdal-3.6.3-h32fa94e_6 requires libjpeg-turbo >=2.1.5.1,<3.0a0, but none of the providers can be installed\\n', 'osx_64_numpy1.21python3.9.____cpython: Encountered problems while solving:\\n  - package libgdal-3.6.3-h32fa94e_6 requires libjpeg-turbo >=2.1.5.1,<3.0a0, but none of the providers can be installed\\n', 'osx_64_numpy1.23python3.11.____cpython: Encountered problems while solving:\\n  - package libgdal-3.6.3-h32fa94e_6 requires libjpeg-turbo >=2.1.5.1,<3.0a0, but none of the providers can be installed\\n']"
    },
    "pyproj": {
      "num_descendants": 1,
      "immediate_children": [
        "qgis"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/pyproj-feedstock/pull/135",
      "pr_status": "unknown"
    },
    "qgis": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/qgis-feedstock/pull/319",
      "pr_status": "unknown"
    },
    "r-proj4": {
      "num_descendants": 1,
      "immediate_children": [
        "r-gribr"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/r-proj4-feedstock/pull/40",
      "pr_status": "unstable"
    },
    "r-rgdal": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4779824768\">bot CI job</a>): main: ['linux_64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.0-h9e01966_0 requires pcre2 >=10.36,<10.37.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.0-h4d23a9d_0 requires pcre2 >=10.36,<10.37.0a0, but none of the providers can be installed\\n']"
    },
    "r-sf": {
      "num_descendants": 1,
      "immediate_children": [
        "r-lwgeom"
      ],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4780263150\">bot CI job</a>): main: ['linux_64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.3-h2f963a2_5 requires jpeg >=9e,<10a, but none of the providers can be installed\\n', 'linux_aarch64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.3-ha74fc48_5 requires jpeg >=9e,<10a, but none of the providers can be installed\\n', 'linux_ppc64le_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.3-h0339487_4 requires jpeg >=9e,<10a, but none of the providers can be installed\\n', 'osx_64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.3-h490d416_4 requires jpeg >=9e,<10a, but none of the providers can be installed\\n']"
    },
    "rasterio": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/rasterio-feedstock/pull/264",
      "pr_status": "unstable"
    },
    "vtk": {
      "num_descendants": 1,
      "immediate_children": [
        "visan"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/vtk-feedstock/pull/292",
      "pr_status": "clean"
    },
    "r-gribr": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "r-lwgeom": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "magics-metview": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/magics-metview-feedstock/pull/64",
      "pr_status": "unstable"
    },
    "magics-metview-batch": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/magics-metview-batch-feedstock/pull/55",
      "pr_status": "unstable"
    },
    "fimex": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/fimex-feedstock/pull/55",
      "pr_status": "unstable"
    },
    "r-gdalcubes": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4780263150\">bot CI job</a>): main: ['linux_64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.3-h2f963a2_5 requires jpeg >=9e,<10a, but none of the providers can be installed\\n', 'linux_64_r_base4.2: Encountered problems while solving:\\n  - package libgdal-3.6.3-h0e7c1b1_3 requires pcre2 >=10.40,<10.41.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.3-h490d416_4 requires jpeg >=9e,<10a, but none of the providers can be installed\\n', 'osx_64_r_base4.2: Encountered problems while solving:\\n  - package libgdal-3.6.3-h32fa94e_6 requires pcre2 >=10.40,<10.41.0a0, but none of the providers can be installed\\n']"
    },
    "r-terra": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4779824768\">bot CI job</a>): main: ['linux_64_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.6.3-h0e7c1b1_3 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_64_r_base4.2: Encountered problems while solving:\\n  - package libgdal-3.6.3-h0e7c1b1_3 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_aarch64_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.6.3-h09f3c6d_3 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_aarch64_r_base4.2: Encountered problems while solving:\\n  - package libgdal-3.6.3-h09f3c6d_3 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_ppc64le_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.6.3-h1089159_8 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_ppc64le_r_base4.2: Encountered problems while solving:\\n  - package libgdal-3.6.3-h1089159_8 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.6.3-h32fa94e_6 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.2: Encountered problems while solving:\\n  - package libgdal-3.6.3-h32fa94e_6 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n']"
    },
    "visan": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "mapserver": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/mapserver-feedstock/pull/25",
      "pr_status": "unknown"
    },
    "pygplates": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/pygplates-feedstock/pull/5",
      "pr_status": "unknown"
    }
  }
}